src/GafferScene/RenderController.cpp

```diff
@@ -1419,6 +1427,37 @@ size_t RenderController::getMinimumExpansionDepth() const
 	return m_minimumExpansionDepth;
 }
 
+void RenderController::setManifestRequired( bool manifestRequired )
+{
+	if( manifestRequired == m_manifestRequired )
+	{
+		return;
+	}
+
+	m_manifestRequired = manifestRequired;
+
+	// We apply the manifest update immediately. This allows a client who calls
+	// setManifestRequired( true ) to assume that subsequent calls to renderManifest()
+	// will not return nullptr.
+	if( !manifestRequired )
+	{
+		m_renderManifest.reset();
```

Comment:
Do we really want to reset the manifest here? We might have an ID output that requires us to keep it.